### PR TITLE
Feat(orders) Allow tomte orders to be submitted

### PR DESCRIPTION
**Pull request by Vince-janv**
### Changed

- Allows orders with data_analyses="tomte" to be ordered

### This [version](https://semver.org/) is a
- [ ] **MINOR** - when you add functionality in a backwards compatible manner

### CODE REVIEW

File: tests/meta/orders/test_meta_orders_status.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 from cg.meta.orders.rml_submitter import RmlSubmitter
 from cg.meta.orders.sars_cov_2_submitter import SarsCov2Submitter
 from cg.meta.orders.submitter import Submitter
+from cg.meta.orders.tomte_submitter import TomteSubmitter
```
Comment (diitaz93) [inline tests/meta/orders/test_meta_orders_status.py:21]: is this actually used?

**Review by diitaz93 — APPROVED** [review]
LGTM 👍

File: tests/meta/orders/test_meta_orders_status.py (hunk 1 above)
Comment (islean) [reply tests/meta/orders/test_meta_orders_status.py]: Nope, good catch!

**Comment by islean** [comment]
Deployed to `cg-services-prod`

**Pull request merged**